doc/api.yml

```diff
@@ -1819,17 +1819,19 @@ components:
     UserIdentifier:
       type: object
       properties:
-        id:
-          type: string
-        ref:
-          type: string
-          example: STD000001
-        first_name:
-          type: string
-        last_name:
-          type: string
-        email:
-          type: string
+         id:
+           type: string
+         ref:
+           type: string
+           example: STD000001
+         first_name:
+           type: string
+         last_name:
+           type: string
+         email:
+           type: string
+         cin:
+           type: string
```

Comment:
looks like you added a bit of space on all attributes.

also NIC is National Identity Card so use NIC